<launch>

    <rosparam file="$(find exo_angle_control)/config/control_config.yaml" command="load" />

    <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false"
        output="screen" ns="/rrbot"
        args="  joint_state_controller
                joint1_position_controller
                joint2_position_controller" />

    <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher"
    respawn="false" output="screen">
<remap from="/joint_states" to="/rrbot/joint_states" />
    </node>
    <!-- <node name="anglePubNode" pkg="exo_angle_control" type="anglePubSub" ns="oneArduinoNs"/> -->
    <!-- <node name="ROScontrol" pkg="exo_angle_control" type="ROScontrol1" ns="oneArduinoNs"/> -->

    <!-- <node name="arduinoSync0" pkg="rosserial_python" type="serial_node.py" ns="oneArduinoNs"
    args="/dev/ttyACM0"/> -->

    <!-- <node name="arduinoSync1" pkg="rosserial_python" type="serial_node.py" ns="oneArduinoNs"
    args="/dev/ttyACM1"/> -->

    <!-- <node name="arduinoSync2" pkg="rosserial_python" type="serial_node.py" ns="oneArduinoNs"
    args="/dev/ttyACM2"/> -->

    <!-- <node name="arduinoSync3" pkg="rosserial_python" type="serial_node.py" ns="oneArduinoNs"
    args="/dev/ttyACM3"/> -->


    <!-- <node name="slider" pkg="rqt_ez_publisher" type="rqt_ez_publisher" ns="oneArduinoNs"/> -->

</launch>